devops/openshift/db-migrations-repl.yml

```diff
@@ -0,0 +1,102 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: ${NAME}
+labels:
+  project: ${PROJECT}
+  service: ${SERVICE_NAME}
+objects:
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      labels:
+        app: ${NAME}
+      name: ${NAME}
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: ${NAME}
+      strategy:
+        type: RollingUpdate
+      template:
+        metadata:
+          labels:
+            app: ${NAME}
+        spec:
+          containers:
+            - env:
+                - name: VERSION
+                  value: ${BUILD_TAG}
+                - name: NODE_ENV
+                  value: ${NODE_ENV}
+                - name: POSTGRES_HOST
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${DB_SERVICE_KEY}
+                      name: ${DB_SECRET_NAME}
+                - name: POSTGRES_USER
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${DB_USERNAME_KEY}
+                      name: ${DB_SECRET_NAME}
+                - name: POSTGRES_PASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${DB_PASSWORD_KEY}
+                      name: ${DB_SECRET_NAME}
+                - name: POSTGRES_DB
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${SIMS_DB_NAME}
+                      name: ${DB_MIGRATIONS_SECRET_NAME}
+              image: ${IMAGE_REGISTRY}/${BUILD_NAMESPACE}/${IMAGE_STREAM_TAG}
+              command: ["npm", "run", "migration:repl"]
+              imagePullPolicy: Always
+              name: ${NAME}
+              tty: true
+              stdin: true
+              resources:
+                limits:
+                  cpu: ${CPU_LIMIT}
+                  memory: ${MEMORY_LIMIT}
+                requests:
+                  cpu: ${CPU_REQUEST}
+                  memory: ${MEMORY_REQUEST}
+parameters:
+  - name: NAME
+    required: true
+  - name: PROJECT
+    value: sims
+  - name: SERVICE_NAME
+    value: db-migrations-repl
+  - name: CPU_LIMIT
+    value: "0.5"
+  - name: MEMORY_LIMIT
+    value: "512M"
+  - name: CPU_REQUEST
+    value: "0.5"
+  - name: MEMORY_REQUEST
+    value: "512M"
```

Comment:
This seems to be half the memory of what we have been using for DB migrations. But still works fine right?
![image](https://github.com/user-attachments/assets/e9a7b0bc-bbad-43dd-ad33-c9fdd8201cd3)

Reply:
So far it seems good enough. If it is not enough, we can adjust as we go. 
If more resources are needed during a deploy, we can quickly change in the yaml and persist back to the repo.
It seems that most of the resources are required only during the run of the `ts-node`, once the application is running the processing will happen mostly on the DB server.